shared/fs/common/hooks.tsx

```diff
@@ -54,15 +42,15 @@ const useFsNonPathSubscriptionEffect = (topic: RPCTypes.SubscriptionTopic) => {
 
 export const useFsPathMetadata = (path: Types.Path) => {
   useFsPathSubscriptionEffect(path, RPCTypes.PathSubscriptionTopic.stat)
-  const dispatch = useDispatchWhenConnectedAndOnline()
```

Comment:
why this change?

Reply:
In offline mode, we still need this in synced folder. So just need to make sure KBFS is connected.